modules/core/portal-bootstrap/build.gradle

```diff
@@ -76,13 +76,15 @@ dependencies {
 	provided group: "biz.aQute.bnd", name: "biz.aQute.bndlib", version: "3.1.0"
 	provided group: "com.liferay", name: "com.liferay.osgi.felix.util", version: "1.0.0"
 	provided group: "com.liferay", name: "com.liferay.osgi.service.tracker.collections", version: "2.0.0"
+	//provided group: "com.liferay", name: "com.liferay.messaging.api", version: "1.0.0"
```

Comment:
You should use this, just make sure to build the api module first and including the `installCache` target in the gradle command you invoke.